azure-pipelines.yml

```diff
@@ -31,7 +31,7 @@ resources:
     endpoint: build
 
 stages:
-- stage: Build
+- stage: BuildVS
```

Comment:
>BuildVS [](http://example.com/codeflow?start=9&length=7)

I think you extracted vs from below section, what is the motivation? I see previous code simpler.